py/scheduler.c

```diff
@@ -146,6 +169,33 @@ bool MICROPY_WRAP_MP_SCHED_SCHEDULE(mp_sched_schedule)(mp_obj_t function, mp_obj
     return ret;
 }
 
+#if MICROPY_SCHEDULER_NODES
+bool mp_sched_schedule_node(mp_sched_node_t *node, mp_sched_callback_t callback) {
```

Comment:
What is the use case for the return value?

Reply:
I didn't think of any use case yet, I just thought it was potentially useful, and mirrors the return value of `mp_sched_schedule()`.

If you think it's better to remove it for now I'm happy to do that.  It would save code size.

Reply:
I left this in, it might be useful.